### PR TITLE
add todos and update run.sh

### PR DESCRIPTION
**Pull request by Trotyl15**

### CODE REVIEW

File: imaging/imaging.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 CAMERA_WIDTH = 1920
 CAMERA_HEIGHT = 1200
 
-SAVE_PREFIX = "log_record_" + str(int(time.time())) + "_"
+SAVE_PREFIX = "images/log_record_" + str(int(time.time())) + "_"
```
Comment (Xierumeng) [inline imaging/imaging.py:15]: Is this successful if the directory doesn't exist?

File: imaging/rc.local.copyme [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 # Copy the following contents to the end of rc.local:
 
-chmod +x /home/warg/model-training/imaging/run.sh
-/home/warg/model-training/imaging/run.sh
+chmod +x /home/warg/model-training/imaging/run.sh # TODO: Update
```
Comment (Xierumeng) [inline imaging/rc.local.copyme]: Why does this need updating?

Comment (Xierumeng) [inline imaging/rc.local.copyme]: 2 spaces between end of statement and `#` .

File: imaging/run.sh [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
-v4l2-ctl -d /dev/video0 --set-ctrl gain=0
 v4l2-ctl -d /dev/video0 --set-ctrl white_balance_temperature=4600
 v4l2-ctl -d /dev/video0 --set-ctrl white_balance_automatic=1
 v4l2-ctl -d /dev/video0 --set-ctrl auto_exposure=3
 
 # OpenCV Python
-touch /home/warg/record.log  # To confirm script has run
-cd /home/warg/model-training/
+touch /home/warg/record.log  # To confirm script has run # TODO: Update
+cd /home/warg/model-training/ # TODO: Update
```
Comment (Xierumeng) [inline imaging/run.sh]: Again, why with the TODOs?

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

File: imaging/imaging.py (hunk 1 above)
Comment (Trotyl15) [reply imaging/imaging.py:15]: No, but I've added the directory with a placeholder file to the repo.

File: imaging/rc.local.copyme (hunk 2 above)
Comment (Trotyl15) [reply imaging/rc.local.copyme]: To make sure people change `warg` to `warg-rpi4b`?

File: images/.placeholder
Comment (Xierumeng) [inline images/.placeholder:1]: Add text explaining why this file exists.

File: imaging/rc.local.copyme [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 # Copy the following contents to the end of rc.local:
 
-chmod +x /home/warg/model-training/imaging/run.sh
-/home/warg/model-training/imaging/run.sh
+chmod +x /home/warg/model-training/imaging/run.sh  # TODO: Update
```
Comment (Xierumeng) [inline imaging/rc.local.copyme]: Make the comment clearer of what needs to be done.

File: imaging/run.sh [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
 # Camera calibration
-v4l2-ctl -d /dev/video0 --set-ctrl gain=0
 v4l2-ctl -d /dev/video0 --set-ctrl white_balance_temperature=4600
 v4l2-ctl -d /dev/video0 --set-ctrl white_balance_automatic=1
 v4l2-ctl -d /dev/video0 --set-ctrl auto_exposure=3
 
 # OpenCV Python
-touch /home/warg/record.log  # To confirm script has run
-cd /home/warg/model-training/
+touch /home/warg/record.log  # To confirm script has run  # TODO: Update
```
Comment (Xierumeng) [inline imaging/run.sh]: Same as above.

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

File: images/.placeholder [hunk 6]
```diff
@@ -0,0 +1 @@
+# The placeholder file to keep the "images" directory exists
```
Comment (Xierumeng) [inline images/.placeholder]: `This file ensures that the `images` directory exists.`

Comment (Xierumeng) [inline images/.placeholder]: Add empty line at end of file.

File: imaging/rc.local.copyme [hunk 7]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 # Copy the following contents to the end of rc.local:
 
-chmod +x /home/warg/model-training/imaging/run.sh
-/home/warg/model-training/imaging/run.sh
+chmod +x /home/warg/model-training/imaging/run.sh  # TODO: Update to actual path
```
Comment (Xierumeng) [inline imaging/rc.local.copyme]: Remove word `TODO` . Same with other comments.

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

**Review by Xierumeng — APPROVED** [review]
Approved.

File: images/.placeholder [hunk 8]
```diff
@@ -0,0 +1 @@
+This file ensures that the `images` directory exists.
```
Comment (Xierumeng) [inline images/.placeholder:1]: Nit: `images/` .

**Pull request merged**